package.json

```diff
@@ -6,9 +6,9 @@
   "main": "dist/index.js",
   "scripts": {
     "start": "tsc && electron-forge start",
-    "package": "electron-forge package",
-    "make": "electron-forge make",
-    "publish": "electron-forge publish",
+    "package": "tsc && electron-forge package",
+    "make": "tsc && electron-forge make",
+    "publish": "tsc && electron-forge publish",
```

Comment:
I would actually suggest erasing the out folder (I had some issues where after the transpilation with tsc the js files were not updated).

Maybe we can introduces some `clean` script with `rimraf out`